installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc

```diff
@@ -127,3 +127,4 @@ include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 * xref:../../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks[Enabling multipathing with kernel arguments on {op-system}].
 * xref:../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
 * If the mirror registry that you used to install your cluster has a trusted CA, add it to the cluster by xref:../../openshift_images/image-configuration.adoc#images-configuration-cas_image-configuration[configuring additional trust stores].
+* The link:https://github.com/ocp-power-automation/rsct-operator[RSCT Operator] can be installed to enable RSCT on the cluster.
```

Comment:
ditto

Reply:
I'm fine with this as is, but just something to consider if you wind up pushing another revision